security/stunnel/src/opnsense/mvc/app/models/OPNsense/Stunnel/Stunnel.xml

```diff
@@ -81,6 +81,10 @@
                     <Type>cert</Type>
                     <ValidationMessage>Please select a valid certificate from the list</ValidationMessage>
                 </servercert>
+                <chainIntermediateCAs type="BooleanField">
+                    <default>0</default>
+                    <Required>Y</Required>
```

Comment:
You need to increase the model number in this file (see above)

Reply:
Do you mean changing `<model><version/>`? to `1.0.4` ok?

Reply:
yep, thanks